src/mrack/providers/beaker.py

```diff
@@ -112,9 +113,8 @@ def _req_to_bkr_job(self, req):
         specs.update({"priority": "Normal"})
 
         # Add allowed keys
-        keypair = self.prov_config.get("keypair", None)
-        if keypair:
-            specs.update({"ks_append": self._allow_ssh_key(keypair)})
+        if self.keypair:
```

Comment:
`keypair` is listed in the `_required_config_attrs` property for Beaker, is this conditional still necessary for some reason?

Reply:
eh no you are correct.